Doc/library/decimal.rst

```diff
@@ -367,6 +367,8 @@ Decimal objects
    appears above.  These include decimal digits from various other
    alphabets (for example, Arabic-Indic and Devanāgarī digits) along
    with the fullwidth digits ``'\uff10'`` through ``'\uff19'``.
+   The characters in the strings accepted for ``infinity`` and ``nan``
+   may be in any case.
```

Comment:
```suggestion
   Case is not significant, so, for example, ``inf``, ``Inf``, ``INFINITY``,
   and ``iNfINity`` are all acceptable spellings for positive infinity.
```
This is better (following to the float docs)?

Reply:
That looks good to me!